### PR TITLE
Update for person = null and redirects need return

### PR DESCRIPTION
**Pull request by jcubbage**
Corrects for error Jason was seeing.

### CODE REVIEW

File: Keas.Mvc/Controllers/ConfirmController.cs [hunk 1]
```diff
@@ -23,20 +23,28 @@ public ConfirmController(ApplicationDbContext context, ISecurityService _securit
         }
 
         public async Task<IActionResult> MyStuff()
-        {
+        {   
             var person = await _securityService.GetPerson(Team);
+            if(person==null){
+                 Message = "You are not yet added to the system.";
+                return RedirectToAction(nameof(HomeController.Index));
```
Comment (jSylvestre) [inline Keas.Mvc/Controllers/ConfirmController.cs]: @jcubbage Need controller name,

File: Keas.Mvc/Controllers/ConfirmController.cs [hunk 2]
```diff
@@ -23,20 +23,28 @@ public ConfirmController(ApplicationDbContext context, ISecurityService _securit
         }
 
         public async Task<IActionResult> MyStuff()
-        {
+        {   
             var person = await _securityService.GetPerson(Team);
+            if(person==null){
```
Comment (srkirkland) [inline Keas.Mvc/Controllers/ConfirmController.cs]: Spaces in the operator here please

File: Keas.Mvc/Controllers/ConfirmController.cs [hunk 3]
```diff
@@ -23,20 +23,28 @@ public ConfirmController(ApplicationDbContext context, ISecurityService _securit
...
+                 Message = "You are not yet added to the system.";
+                return RedirectToAction("Index","Home");
```
Comment (srkirkland) [inline Keas.Mvc/Controllers/ConfirmController.cs]: Interesting, we probably want to make this a much nicer experience in the future, maybe a page with info on what to do.

**Comment by jcubbage** [comment]
Better?

**Pull request merged**